### PR TITLE
Qualty cade challenge

### PR DESCRIPTION
**Pull request by haroldmud**

### CODE REVIEW

File: NumberFormat.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+var numberFormat = function (number) {
+  return number.toLocaleString()
+};
```
Comment (IbrahimBagalwa) [inline NumberFormat.js:3]: you could use the new Es6 Syntax
```js
const numberFormat = (number)=>number.toLocaleString()
```

File: Roman_Numerals_Decoder.jst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    let value = romanObject[roman[i]];
+    if( value >= previousValue){
+      result += value;
+    }else{
+      result -= value;
+    }
+    previousValue = value;
+  }
+	return result;
+}
```
Comment (IbrahimBagalwa) [inline Roman_Numerals_Decoder.jst:23]: Can you change the extension of your file because this is not a valid extension file.

**Review by clever-cr — APPROVED** [review]

File: string_addition.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+function joinStrings(string1, string2){
+  return `${string1} ${string2}`
+}
```
Comment (IbrahimBagalwa) [inline string_addition.js:3]: Could you use ES6 here to make it easy.
```js
const joinStrings=(string1, string2)=> `${string1} ${string2}`

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
Great Job @haroldmud; 
I left few comment to improve the base code

File: Create Phone Number [hunk 4]
```diff
@@ -0,0 +1,9 @@
+function createPhoneNumber(numbers){
+  let phoneNumberForm = '(***) ***-****';
+  
+  for(let i=0; i< numbers.length; i++){
+    phoneNumberForm = phoneNumberForm.replace("*", numbers[i]);
+  }
```
Comment (musayann) [inline Create Phone Number:6]: use array functions

File: roundRobin [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    for(let i = 0 ; i <= jobs.length; i++){
+      if(jobs[i] >= 0){
+      result += jobs[i] <= slice ? jobs[i] : slice
+      jobs[i] = jobs[i]-slice 
+      }
+      if(jobs[index] <= 0) break;
+    }
+  }
+  return result
+}
```
Comment (IbrahimBagalwa) [inline roundRobin:13]: Here as well change the extension file

**Pull request merged**